### PR TITLE
[BREAKING] Refactor: remove voting

### PR DESCRIPTION
**Pull request by sander2**
Breaking changes:
- remove many storage items from staked-relayers
    - stake is now stored in `Stakes`
- removed many dispatchable functions 
- changed `RegisterStakedRelayer` event

### CODE REVIEW

File: crates/staked-relayers/src/lib.rs [hunk 1]
```diff
@@ -128,8 +105,21 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        /// Upgrade the runtime depending on the current `StorageVersion`.
         fn on_runtime_upgrade() -> Weight {
-            Self::_on_runtime_upgrade().expect("runtime upgrade failed");
+            use frame_support::{migration::StorageKeyIterator, Blake2_128Concat};
+
+            StorageKeyIterator::<T::AccountId, StakedRelayer<DOT<T>, T::BlockNumber>, Blake2_128Concat>::new(<Stakes<T>>::module_prefix(), b"ActiveStakedRelayers")
```
Comment (sander2) [inline crates/staked-relayers/src/lib.rs:112]: Please double check

Comment (gregdhill) [reply crates/staked-relayers/src/lib.rs:112]: Seems fine to me, but maybe we should add a small unit test?

File: crates/staked-relayers/src/lib.rs [hunk 2]
```diff
@@ -514,214 +331,12 @@ decl_module! {
...
-    ///
-    /// * `id` - id of the `StatusUpdate`
-    /// * `status_update` - `StatusUpdate` to evaluate
-    fn _evaluate_status_update(
-        id: StatusUpdateId,
-        mut status_update: &mut StatusUpdate<T::AccountId, T::BlockNumber, DOT<T>>,
-    ) -> Result<(), DispatchError> {
-        if status_update.tally.is_approved() {
-            Self::execute_status_update(id, &mut status_update)?;
-            Self::update_sla_score_for_status_update(&status_update, true)?;
```
Comment (gregdhill) [inline crates/staked-relayers/src/lib.rs:665]: Maybe we should also remove the associated logic in SLA for voting?

File: crates/staked-relayers/src/lib.rs (hunk 1 above)
Comment (sander2) [reply crates/staked-relayers/src/lib.rs:112]: How would you write a unit test for this?

Comment (gregdhill) [reply crates/staked-relayers/src/lib.rs:112]: Insert the previous key / value formats into storage and check the iterator correctly updates.

**Review by gregdhill — APPROVED** [review]

**Review by gregdhill — APPROVED** [review]

**Pull request merged**